craft_providers/lxd/lxd.py

```diff
@@ -76,28 +83,49 @@ def is_supported_version(self) -> bool:
         minimum_version = packaging.version.parse(self.minimum_required_version)
         version = self.version()
 
-        if "." not in version:
-            raise LXDError(
-                "Failed to parse LXD version.",
-                details=f"Version data returned: {version!r}",
+        try:
+            parsed_version = packaging.version.parse(version)
+        except packaging.version.InvalidVersion:
+            logger.warning(
+                f"Unknown LXD version. Assuming supported. {version=}"
             )
+            return True
 
-        return packaging.version.parse(version) >= minimum_version
+        return parsed_version >= minimum_version
 
     def version(self) -> str:
         """Query LXD version.
 
         The version is of the format:
-        <major>.<minor>[.<micro>] [LTS]
+        <major>.<minor>[.<micro>]
 
         Version examples:
-        - 5.21.0 LTS
```

Comment:
Couldn't this format get returned if it falls back to `return self._version_cmd()`?

Reply:
Yes, good point. I was originally going to just use pylxd, but if the lxd command is installed but lxd isn't running that'd be a regression.